log/Logger.js

```diff
@@ -30,19 +61,20 @@ const customMessageFormat = format((info) => {
  * }
  */
 const logger = createLogger({
-  level: process.env.LOG_LEVEL || 'info',
+  level: logLevel(),
```

Comment:
You are missing the `levels` property I believe in this configuration, it is used to explicit define custom levels. Otherwise winston is still running with the NPM levels by default, you are not seeing it because your custom log levels are currently a subset of NPM's ones, but if you were to introduce a custom log level in your map, then it wouldn't work

Reply:
added